src/Codeception/Module/Yii2.php

```diff
@@ -804,6 +804,15 @@ public function grabLastSentEmail()
         return end($messages);
     }
 
+    /**
+     * Deletes all sent emails.
+     * @part email
+     */
+    public function deleteSentEmails()
```

Comment:
This is not needed. The client is a public property in the connector.
Since this is something that is reaching in and changing stuff I don't think we should expose it here.

Reply:
If I remove this function, how would I access the client from my unit test? To my understanding, during `codecept build`, the trait `UnitTesterActions` is created and all methods in all enabled modules are added to it.

Reply:
Ah thank you, now I understand how it works :+1:  Updating the PR